recipe/meta.yaml

```diff
@@ -136,6 +136,8 @@ about:
     - library_licenses/http-LICENSE-MIT
     - library_licenses/httparse-LICENSE-APACHE
     - library_licenses/httparse-LICENSE-MIT
+    - library_licenses/httpdate-LICENSE-APACHE
+    - library_licenses/httpdate-LICENSE-MIT
```

Comment:
Gives me a bit of the willies that we pulled a different novel depdendency from the same source tarball...

Reply:
Yeah we would need a build number bump to include the new license files. So the change above makes sense.